misc/PFS_Abbreviations.tsv

```diff
@@ -38,86 +39,91 @@ DRP	Software	General	Data Reduction Pipeline	Software for data processing. There
 enu	ICS	Actor	Entrance Unit	Software to control Spectrograph components other than Camera Units, such as BIA and slit. LAM has a responsibility for its development.
-ETC	PFI	Subcomponent	Electronics Thermal Control	PFI assembly consisting of heat exchangers and coolant lines in order to limit heat dissipation into the telecscope dome air.
+ENU-CR	SpS	Subcomponent	Entrance Unit Control Rack	Spectrograph component. The rack accommodates power supplies, environment controller and so on. Each module has one rack.
+ETC	PFI	Subcomponent	Electronics Thermal Control	PFI assembly consisting of heat exchangers and coolant lines in order to limit heat dissipation into the telescope dome air.
 ETC	SPT	misc	Exposure Time Calculator	Software to calculate exposure time at a given observational configuration. Based on C script by C. Hirata, a python emulator is provided. ETC also simulates resultant spectra.
 ETH	PFI	Subcomponent	Ethernet Hub	Ethernet Hub switch accommodated in Auxiliary Power/Control Electronics (APC) box.
 ETS	Software	General	Exposure Targeting Software	Software to allocate fibers to a provided target list. MPA has a responsibility for its development.
 FAA	PFI/FOCCoS	Subcomponent	Fiber Arm Assembly	Assembly for Cable C, PFI. FAA connects science fiber with microlens and fiber positioner Cobra.
-FBS	PFI	Subcomponent	Fiducial Fiber Beam Spliter	Optics element for fiducial fiber illuminator. FBS is used to reflect LED light into fiducial fibers, and to transmit light from fiducial fibers to their viewing camera. 
+FBS	PFI	Subcomponent	Fiducial Fiber Beam Splitter	Optics element for fiducial fiber illuminator. FBS is used to reflect LED light into fiducial fibers, and to transmit light from fiducial fibers to their viewing camera. 
 FCA	SpS	Subcomponent	Fiber Cable A Assembly	SpS assembly of fiber slit and connectors with Cable B.
 fcc	ICS	Actor	Field Center Camera	Software to control field center camera of PFI. ASIAA has a responsibility for its development.
 FEL	PFI	Subcomponent	Field Element	PFI component. In order to have the similar optical path to HSC, a glass plate of 54 mm thick is accommodated at the bottom of PFI.
 FEM	PFI	Subcomponent	Field Element Mounts	PFI component to mount the field element (FEL) to the Cobra bench (COB). CIT has a responsibility for its development.
 FFC	PFI	Subcomponent	Fixed Fiducial Camera	PFI component to take image of targets acquired on fixed fiducial fibers. ASIAA has a responsibility for its development.
 FFF	PFI	Subcomponent	Fixed Fiducial Fibers	Fibers fixed on the Cobra bench (COB) used as a reference of fiber position. There are 96 FFF in total, 84 for science fibers, and 12 for AG cameras.
+FFI	PFI	Subcomponent	Fiducial Fiber Illuminator	PFI component to back-illuminate fixed fiducial fibers and take image of the light coming through the fiducial fibers.
```

Comment:
with adding this, need to remove FIS?

Reply:
FFI is layer 3, and FIS is layer 4.
I tried to include abbreviations for down to layer 4, for PFI...